modules/authorization-manager/src/test/java/org/opencastproject/authorization/xacml/manager/endpoint/OsgiAclServiceRestEndpointTest.java

```diff
@@ -74,162 +76,183 @@ public void tearDownTest() {
-    .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
+        .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
   }
 
   @Test
   public void testAclExtendInputEmptyRoleExpectsBadRequest() {
     given().formParam("acl", publicAcl).formParam("action", "write").formParam("role", "").expect()
-    .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
+        .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
   }
 
   @Test
   public void testAclExtendInputEmptyActionExpectsBadRequest() {
     given().formParam("acl", publicAcl).formParam("action", "").formParam("role", NEW_ROLE).expect()
-    .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
+        .statusCode(BAD_REQUEST).when().post(host("/acl/extend"));
   }
 
   @Test
   public void testAclReduceInputExistingAceExpectsAceGone() {
     given().formParam("acl", publicAcl).formParam("action", Action.READ.toString())
-    .formParam("role", SERIES_10_INSTRUCTOR_ROLE).expect().body(containsString("{\"ace\":[]}")).statusCode(OK)
-    .when().post(host("/acl/reduce"));
+        .formParam("role", SERIES_10_INSTRUCTOR_ROLE).expect().body(containsString("{\"ace\":[]}")).statusCode(OK)
+        .when().post(host("/acl/reduce"));
   }
 
   @Test
   public void testAclReduceInputWrongRoleExpectsSameAcl() {
     given().formParam("acl", publicAcl).formParam("action", Action.READ.toString()).formParam("role", NEW_ROLE)
-    .expect()
-    .body("ace[0].role", equalTo(SERIES_10_INSTRUCTOR_ROLE))
-    .body("ace[0].action", equalTo(Action.READ.toString()))
-    .body("ace[0].allow", equalTo(true))
-    .statusCode(OK).when().post(host("/acl/reduce"));
+        .expect()
+        .body("ace[0].role", equalTo(SERIES_10_INSTRUCTOR_ROLE))
+        .body("ace[0].action", equalTo(Action.READ.toString()))
+        .body("ace[0].allow", equalTo(true))
+        .statusCode(OK).when().post(host("/acl/reduce"));
   }
 
   @Test
   public void testAclReduceInputWrongActionExpectsSameAcl() {
-    given().formParam("acl", publicAcl).formParam("action", Action.WRITE.toString()).formParam("role", SERIES_10_INSTRUCTOR_ROLE)
-    .expect()
-    .body("ace[0].role", equalTo(SERIES_10_INSTRUCTOR_ROLE))
-    .body("ace[0].action", equalTo(Action.READ.toString()))
-    .body("ace[0].allow", equalTo(true))
-    .statusCode(OK).when().post(host("/acl/reduce"));
+    given()
+        .formParam("acl", publicAcl)
+        .formParam("action", Action.WRITE.toString())
+        .formParam("role", SERIES_10_INSTRUCTOR_ROLE)
+        .expect()
+        .body("ace[0].role", equalTo(SERIES_10_INSTRUCTOR_ROLE))
+        .body("ace[0].action", equalTo(Action.READ.toString()))
+        .body("ace[0].allow", equalTo(true))
+        .statusCode(OK).when().post(host("/acl/reduce"));
   }
 
   @Test
   public void testAclReduceInputEmptyAclExpectsBadRequest() {
     given().formParam("acl", "").formParam("action", "write").formParam("role", NEW_ROLE).expect()
```

Comment:
This isn't formatted as above.  It would be nice if it were, but I'm guessing there's no obvious way to catch this with the autoformatter.

Reply:
Yeah true, difficult to catch this. I basically just look at the reported violations and also fix anything nearby that looks terrible off. At least the stuff that I notice. My PRs don't aim to be perfect, but as long as everyone agrees that they clearly improve the state of the code base, they are probably worth it.